src/cli/init.rs

```diff
@@ -53,6 +53,10 @@ pub struct Args {
     // BREAK (0.27.0): Remove this option from the cli in favor of the `format` option.
     #[arg(long, conflicts_with_all = ["env_file", "format"], alias = "pyproject", hide = true)]
     pub pyproject_toml: bool,
+
+    /// Source Control Management used for this project
+    #[arg(short = 's', long = "scm", default_value = "github")]
+    pub scm: Option<String>,
```

Comment:
Just use the Enum directly for autocompletion, also make sure it works with lowercase and uppercase version of the strings through a test.

Reply:
I'll definitely do that.